crates/pixi_cli/src/build.rs

```diff
@@ -46,6 +47,50 @@ pub struct Args {
     /// Whether to clean the build directory before building.
     #[clap(long, short)]
     pub clean: bool,
+
+    /// The path to `package.xml`, `recipe.yaml`, or `pixi.toml` that can be built.
+    #[arg(long)]
+    pub build_manifest: Option<PathBuf>,
+}
+
+/// Validate that the full path of build manifest exists and is a supported format.
+fn validate_build_manifest(path: &PathBuf) -> miette::Result<()> {
+    if path.is_dir() {
+        miette::bail!(
+            "the build manifest path '{}' is a directory, please provide the path to the manifest file",
+            path.display()
```

Comment:
Could you add a `help` with the possible manifests, which we should keep in the consts.
e.g.:
```
help: the valid manifest files are: pixi.toml, pyproject.toml, package.xml
```